src/der.rs

```diff
@@ -178,10 +191,13 @@ pub fn positive_integer<'a>(input: &mut untrusted::Reader<'a>)
 
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
+    use std;
+    use std::io::BufRead;
     use error;
     use super::*;
     use untrusted;
+    use rustc_serialize::base64::FromBase64;
```

Comment:
By the way, what's your preference on sorting things like `super` or `std` separately from stuff in this crate or other crates like `untrusted` or `signature?